parsec/stubs/marshmallow/__init__.pyi

```diff
@@ -32,33 +32,41 @@ post_dump: Callable[[T], T]
 class MarshmallowSchema:
     def __init__(
         self,
-        only=(),
-        exclude=(),
-        prefix="",
-        strict=None,
-        many=False,
-        context=None,
-        load_only=(),
-        dump_only=(),
-        partial=False,
+        only: tuple[object, ...] = (),
+        exclude: object = (),
+        prefix: str = "",
+        strict: object | None = None,
+        many: bool = False,
+        context: object | None = None,
+        load_only: Iterable[object] = (),
+        dump_only: Iterable[object] = (),
+        partial: bool = False,
     ): ...
-    def dump(self, obj): ...
-    def dumps(self, obj, *args, **kwargs): ...
-    def load(self, data): ...
-    def loads(self, data): ...
+    def dump(self, obj: object) -> object: ...
+    def dumps(self, obj: object, *args: object, **kwargs: object) -> object: ...
+    def load(self, data: object) -> object: ...
+    def loads(self, data: object) -> object: ...
 
     _declared_fields: Dict[str, Field[Any]]
 
 class Schema:
```

Comment:
Can also be made generic
but maybe in another PR :thinking:

Reply:
Yes, this is possible to do. I would rather do it in another PR because this may involve a lot of changes across differents modules

Reply:
I've created the issue for that https://github.com/Scille/parsec-cloud/issues/3592